templates/emails/plain/email-order-items.php

```diff
@@ -22,9 +22,15 @@
 foreach ( $items as $item_id => $item ) :
 	if ( apply_filters( 'woocommerce_order_item_visible', true, $item ) ) {
 		$product = $item->get_product();
+		
```

Comment:
Please, declare those variables here first, so we don't need to check with `! empty()`, this will have the same behavior of the HTML template.

Thanks for your help!